assets/src/styles/blocks/CarouselHeader/CarouselHeaderStyle.scss

```diff
@@ -78,6 +78,14 @@ $medium-image-height: 600px;
 .carousel-header {
   position: relative;
   width: 100vw;
+
+  &.block {
+    display: block;
+  }
+
+  &.block-header {
+    margin-bottom: 0;
```

Comment:
This value reset the default one from the [block styles](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/layout/_blocks.scss#L15-L18) which is causing the next layout issue.

![Screenshot 2022-07-21 at 12 15 54](https://user-images.githubusercontent.com/77975803/180250653-a2952e29-5480-4815-ab65-2b3d05b7116e.png)